Python_based_scripts/Extracting_Water_Temperature_at_Site.ipynb

```diff
@@ -0,0 +1,1188 @@
+    "  \n",
+    "This notebook demonstrates direct access to the [AIMS Temperature Logger Monitoring Program](https://apps.aims.gov.au/metadata/view/4a12a8c0-c573-11dc-b99b-00008a07204e) dataset in the RIMReP `geoparquet` collection. This dataset includes sea temperature data from 1980 for tropical and subtropical coral reefs around Australia, and it includes sites within the Great Barrier Reef.   \n",
+    "  \n",
+    "In this example, we will extract the coordinates for all sites sampled in this monitoring program and then extract the temperature data for a specific site of interest. This means that we will not need to know the exact location of the site, but we will need to know the name of the site of our interest. \n",
+    "  \n",
+    "**Note:** The `geoparquet` collection is a compilation of datasets that have been converted to the Apache `parquet` format. This format is designed for efficient storage and retrieval of tabular data. The `geoparquet` collection is stored on the RIMReP S3 bucket and is accessible to all users."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Loading libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Accessing S3 bucket\n",
+    "from pyarrow import parquet as pq\n",
+    "# Manipulating dataframes\n",
+    "import pandas as pd\n",
+    "# Checking patterns in strings using regular expressions (regex)\n",
+    "import re\n",
+    "# Load and manipulate geospatial data\n",
+    "import dask_geopandas\n",
+    "import geopandas as gpd\n",
+    "# Plotting\n",
+    "import matplotlib.pyplot as plt\n",
+    "#Color maps\n",
+    "from matplotlib import colormaps as cmaps\n",
+    "# Mapping\n",
+    "import cartopy.crs as ccrs\n",
+    "import cartopy.feature as cf"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Inspecting metadata\n",
+    "The AIMS Temperature dataset contains over 150 million records. We will use its metadata to check the structure of the dataset and to determine the best way to access the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<pyarrow._parquet.FileMetaData object at 0x00000262B4EB51C0>\n",
+       "  created_by: parquet-cpp-arrow version 10.0.1\n",
+       "  num_columns: 23\n",
+       "  num_rows: 150663521\n",
+       "  num_row_groups: 50\n",
+       "  format_version: 2.6\n",
+       "  serialized_size: 155051"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "#Connecting to metadata file\n",
+    "md = pq.read_metadata(f\"s3://rimrep-data-public/091-aims-sst/test-50-64-spatialpart/_metadata\")\n",
```

Comment:
is the URL of `rimrep-data-public/091-aims-sst/test-50-64-spatialpart/_metadata` correct? it works for me but I can see it has the old naming of 091-aims-sst in it and i'm concerned that this is an old file path that just happens to be working - I don't have access to S3 otherwise i'd check myself